inorbit_edge/robot.py

```diff
@@ -140,10 +145,50 @@ def on_connect(self, client, userdata, flags, rc):
         # Send robot online status.
         # This method is blocking so do it on a separate thread just in case.
         threading.Thread(
-            target=self.send_robot_status, kwargs={"robot_status": "1"}
+            target=self._send_robot_status, kwargs={"robot_status": "1"}
         ).start()
 
-    def send_robot_status(self, robot_status):
+        # Configure custom command callback if provided
+        if self.custom_command_callback:
+            self.register_custom_command_callback(self.custom_command_callback)
+
+    def _on_message(self, client, userdata, msg):
+        """MQTT client connect callback.
+
+        Args:
+            client:     the client instance for this callback
+            userdata:   the private user data as set in Client() or userdata_set()
+            msg:        an instance of MQTTMessage. This is a class with
+                        members topic, payload, qos, retain.
+        """
+
+        # Parse message and execute custom command callback
+        if self.custom_command_callback:
```

Comment:
We should check that the message is from the right topic, now it will work because you are only subscribing to one topic.